src/lib/ext-monitor.js

```diff
@@ -17,9 +19,15 @@ export default class ExtensionMonitor {
   }
 
   async isActivityLogPageOpen() {
+    const activitylogPage = getActivityLogPageURL();
+    if (this.isDevToolPanelOpen) {
+      return true;
+    }
+
     const tab = await browser.tabs.query({
-      url: getActivityLogPageURL(),
+      url: [activitylogPage, `${activitylogPage}?*`],
```

Comment:
at a first glance this looks unrelated to the main goal of this PR (the activitylog page running in the devtools panel), I guess that this is something you added while you were trying to use the activitylog page with the filterTabId parameter in a tab, is that right?

I guess that this would also make `isActivityLogPageOpen` to resolve to true when there are activitylog pages for the loaded log files opened in tabs, is that the case? is that expected?

Reply:
Yeah, initially, it was for the activitylog page with the filterTabId parameter in a tab. I have updated this part excluding this change.

Reply:
I think we should keep this change, but fixing it by:
```
[activitylogPage, `${activitylogPage}?filterTabId*`],
```
. this will allow us to open tab with any tabId and retrieve realtime logs of that tab.

Reply:
updated :-)